src/js/tracing/timetodisplay.tsx

```diff
@@ -247,25 +248,25 @@ function updateFullDisplaySpan(frameTimestampSeconds: number, passedInitialDispl
   const initialDisplayEndTimestamp = existingInitialDisplaySpan && spanToJSON(existingInitialDisplaySpan).timestamp;
   if (!initialDisplayEndTimestamp) {
     fullDisplayBeforeInitialDisplay.set(activeSpan, true);
-    logger.warn(`[TimeToDisplay] Full display called before initial display for active span.`);
+    logger.warn(`[TimeToDisplay] Full display called before initial display for active span (${activeSpan.spanContext().spanId}).`);
     return;
   }
 
   const span = startTimeToFullDisplaySpan();
   if (!span) {
-    logger.warn(`[TimeToDisplay] No span found or created, possibly performance is disabled.`);
+    logger.warn(`[TimeToDisplay] No TimeToFullDisplay span found or created, possibly performance is disabled.`);
     return;
   }
 
   if (spanToJSON(span).timestamp) {
-    logger.warn(`[TimeToDisplay] ${spanToJSON(span).description} span already ended.`);
+    logger.warn(`[TimeToDisplay] ${spanToJSON(span).description} (${span.spanContext().spanId}) span already ended.`);
```

Comment:
Suggestion: It would be nice to store the result of `spanToJSON` so that we we don't need to call it twice on  on  lines 261, 262 and 263

Reply:
Yeah, I agree, I will change that.